troubleshooting/tools/log-loss-test-framework/logger/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine as build-env
```

Comment:
Interesting choice. I suppose alpine can be used instead of amazon2 linux as long as we don't distribute.